init.lua

```diff
@@ -499,9 +541,7 @@ end
 
 local function get_inv_items(player)
 	local inv = player:get_inventory()
-	local stacks = inv:get_list("main")
-	local craftlist = inv:get_list("craft")
-	stacks = table_merge(stacks, craftlist)
+	local stacks = table_merge(inv:get_list("main"), inv:get_list("craft"))
```

Comment:
Here some localization is needed. I hate it when getters are inside function calls.